docs/src/kubernetes/installing-helm.md

```diff
@@ -0,0 +1,94 @@
+
+
+As part of the Zeebe project, we are providing 3 Zeebe HELM Charts: 
+- **Zeebe Cluster**: Deploys a Zeebe Cluster with 3 brokers using the camunda/zeebe docker image. This Chart depends on ElasticSearch HELM Chart and optionally on Kibana HELM Chart. This chart is hosted in the following repository, where you can find more information about its configuration: [http://github.com/zeebe-io/zeebe-cluster-helm/](http://github.com/zeebe-io/zeebe-cluster-helm/)
+- **Zeebe Operate**: Deploys Zeebe Operate which conects to an existing ElasticSearch. This chart source code can be located here: [http://github.com/zeebe-io/zeebe-operate-helm/](http://github.com/zeebe-io/zeebe-operate-helm/)
+- **Zeebe Full** (Parent): Deploys a Zeebe Cluster + Operate + Ingress Controller. This parent chart can be located here: [http://github.com/zeebe-io/zeebe-full-helm/](http://github.com/zeebe-io/zeebe-full-helm/)
+
+![Charts](zeebe-helm-charts.png)
+
+### Initializing HELM in your Cluster
+
+You need to have `kubectl` already configured against a Kubernetes Cluster to install HELM (server side/tiller) into your cluster. 
+> Here you can download the [helm-service-account-role.yaml](helm-service-account-role.yaml) file
+
+You also need to have the `helm` CLI tool installed as listed in the prerequisites section.
+
+```
+> kubectl apply -f helm-service-account-role.yaml
+> helm init --service-account helm --upgrade 
+```
+
+This install HELM server side components in your cluster and it will enable the `helm` cli tool to install HELM Charts into your environment. 
+
+
+### Add Zeebe HELM Repository
+
+The next step is to add the Zeebe official HELM Chart repository to your installation. Once this is done, HELM will be able to fetch and install Charts hostested in [http://helm.zeebe.io](http://helm.zeebe.io).
+```
+> helm repo add zeebe https://helm.zeebe.io
+> helm repo update
+```
+
+Once this is done, we are ready to install any of the HELM Charts hosted in the official Zeebe HELM Chart repo. 
+
+
+### Install Zeebe Full HELM Chart (Zeebe Cluster + Operate + Ingress Controller)
+
+In this section we are going to install all the available Zeebe components inside a Kubernetes Cluster. Notice that this Kubernetes cluster can have already running services and Zeebe is going to installed just as another set of services. 
+
+```
+> helm install --name <RELEASE NAME> zeebe/zeebe-full
+```
+
+> Note: change <RELEASE NAME> with a name of your choice
+> Notice that you can add the `-n` flag to specify in which Kubernetes namespace the components should be installed.
+
+Installing all the components in a cluster requires all the Docker images to be downloaded to the remote cluster, depending on which Cloud Provider you are using is the amount of time that it will take to fetch all the images. 
```

Comment:
[nitpick]
using is the amount of time that it will take to fetch all the images.
using, the amount of time that it will take to fetch all the images will vary.